### PR TITLE
Full size header

### PR DESCRIPTION
**Pull request by ozasadnyy**
## Mobile

<br/>

### CODE REVIEW

**Comment by sophieH29** [comment]
Suggest to add scroll on click to animated button on the hero's bottom

File: src/pages/home-page.html [hunk 1]
```diff
@@ -18,10 +22,136 @@
 
       :host {
         display: block;
+        height: 100%;
+      }
+
+      hero-block {
+        height: calc(100% + 56px);
```
Comment (unreadableusername) [inline src/pages/home-page.html]: I guess `height: 100vh;` will be better

File: src/pages/home-page.html [hunk 2]
```diff
@@ -18,10 +22,136 @@
...
+          transform: translate(0, 5px)
+        }
+        100% {
+          transform: translate(0, 0)
+        }
+      }
+
+      @media (min-width: 640px) {
+        hero-block {
+          height: calc(100% + 64px);
```
Comment (unreadableusername) [inline src/pages/home-page.html]: same `100vh;`

Comment (ozasadnyy) [reply src/pages/home-page.html]: I wish it works, but mobile Safari + now mobile Chrome doesn't handle it correctly, so I had to add a bunch of `height: 100%` through the app.

Comment (unreadableusername) [reply src/pages/home-page.html]: strange 😢, according to https://caniuse.com/#search=vh , it should work

Comment (ozasadnyy) [reply src/pages/home-page.html]: It is supported but in their way :)
More info here: https://stackoverflow.com/a/37113430

**Comment by ozasadnyy** [comment]
@sophieH29 added

File: src/elements/header-toolbar.html [hunk 3]
```diff
@@ -10,11 +10,24 @@
     <style>
 
       :host {
-        --iron-icon-fill-color: var(--secondary-text-color);
+        --iron-icon-fill-color: currentColor;
```
[the remark this answers is not in the document]
Comment (ozasadnyy) [reply src/elements/header-toolbar.html]: The currentColor keyword represents the value of an element's color property. This lets you use the color value on properties that do not receive it by default.

File: src/elements/hero-block.html [hunk 4]
```diff
@@ -18,19 +11,20 @@
     <style>
 
       :host {
+        margin-top: -56px;
```
[the remark this answers is not in the document]
Comment (ozasadnyy) [reply src/elements/hero-block.html]: To move hero block under the toolbar

**Pull request merged**